src/tests/test_ggplot.py

```diff
@@ -409,6 +427,20 @@ def test_facet_wrap_stacked_histogram_cmap(diamonds_data):
     )
 
 
+@_cleanup_cm()
+@image_comparison(
+    baseline_images=["test_facet_wrap_default_with_nulls"],
+    extensions=["png"],
+    remove_text=False,
+)
+def test_facet_wrap_default_with_nulls(penguins_with_nulls):
```

Comment:
Can u try to use `penguins_data` directly here? I don't think we need an intermediate table since this data already contains nulls

Reply:
You can also try with some dummy data

Reply:
Yes it works, just had to setup ` %sql duckdb://` in `penguins_data`. We don't need to create an intermediate table now.